src/components/DpDataTable/DpDataTable.vue

```diff
@@ -597,31 +649,23 @@ export default {
      * Therefore, we have a normal table, get the auto-sized cell width and set the layout to fixed afterwards.
      */
     if (this.isResizable || this.isTruncatable) {
-      const firstRow = this.tableEl.firstChild
-      const tableHeaders = Array.prototype.slice.call(firstRow.childNodes)
-      tableHeaders.forEach(tableHeader => {
-        /**
-         * Some of childNodes of the first table row are not Element nodes but comments or text.
-         * This originates in the Vue template compiler leaving empty html comments when rendering
-         * falsy `v-if` blocks. We allow only nodeType "Element" to access its `getBoundingClientRect` api.
-         */
-        if(tableHeader.nodeType === 1) {
-          const width = tableHeader.getBoundingClientRect().width
-          tableHeader.style.width = width + 'px'
-        }
-      })
+      const firstRow = this.tableEl.getElementsByTagName('tr')[0]
+      const tableHeaders = firstRow ? firstRow.children : null
```

Comment:
Nitpick on wording: even if you did not introduce the variable name `tableHeaders`... when reading through the code, it seems unintuitive to name it this way (with "tableHeader" i associate the complete header of the table, not the `th` element actually refered here). Maybe we could name the variable `tableHeaderElements`? idk...

Reply:
right, done here https://github.com/demos-europe/demosplan-ui/pull/835/commits/19afd6dc9b016b75745ba7a70f1425433ee8a3d7